<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-6UPTAFLI" failures="0" tests="2" name="liveTicketTestCases.CTMSLiveTicketCheckwithemail" time="0.040" errors="1" timestamp="2023-05-25T11:57:39 IST" skipped="1">
  <testcase classname="liveTicketTestCases.CTMSLiveTicketCheckwithemail" name="findCTMSLiveTickets" time="0.000">
    <skipped/>
  </testcase> <!-- findCTMSLiveTickets -->
  <system-out/>
  <testcase classname="liveTicketTestCases.CTMSLiveTicketCheckwithemail" name="endReport" time="0.040">
    <error message="Cannot attach file &quot;user.dir/Test-Output/ExtentReport/CTMS_Tickets.html&quot;" type="org.apache.commons.mail.EmailException">
      <![CDATA[org.apache.commons.mail.EmailException: Cannot attach file "user.dir/Test-Output/ExtentReport/CTMS_Tickets.html"
at org.apache.commons.mail.MultiPartEmail.attach(MultiPartEmail.java:333)
at liveTicketTestCases.CTMSLiveTicketCheckwithemail.SendEmail(CTMSLiveTicketCheckwithemail.java:242)
at liveTicketTestCases.CTMSLiveTicketCheckwithemail.endReport(CTMSLiveTicketCheckwithemail.java:191)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:619)
at org.testng.TestRunner.afterRun(TestRunner.java:849)
at org.testng.TestRunner.run(TestRunner.java:590)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.IOException: "user.dir/Test-Output/ExtentReport/CTMS_Tickets.html" does not exist
at org.apache.commons.mail.MultiPartEmail.attach(MultiPartEmail.java:322)
... 24 more
]]>
    </error>
  </testcase> <!-- endReport -->
  <system-out/>
</testsuite> <!-- liveTicketTestCases.CTMSLiveTicketCheckwithemail -->
